.github/workflows/unit-tests.yml

```diff
@@ -1,27 +0,0 @@
-name: Unit Tests
-
-on:
-  push:
-  pull_request:
-
-jobs:
-  tests:
-    name: Unit tests
-
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Setup node
-        uses: actions/setup-node@v2
-        with:
-          node-version: 14
```

Comment:
Node version number was bumped to 16. This is still TBD, I need to check the minimum version supported by this package. I want to read it dynamically from `package.json` "engines" so we have unification there.